actions.yaml

```diff
@@ -50,3 +50,8 @@ rebalance:
       description: Only generate the partition rebalance proposals and estimated result, without executing
       type: boolean
       default: true
+    brokerid:
+      description: Broker ID newly added to the cluster or to be removed
```

Comment:
**todo:** Mention here that the broker ID == unit id, with an example:
```
kafka/0 --> 0
```

It's obvious to us, but to a user without any prior knowledge of the charm internal, it might be weird. ZK for example has server ID == unit id + 1.

Reply:
Yes, that's an excellent suggestion, as I got confused myself. Done in 4d373b742e39